components/AnchorNavigation/src/stories/bem.stories.mdx

```diff
@@ -29,7 +29,7 @@ import "../index.scss";
   {/* eslint-disable react/no-unknown-property */}
   <Story name="Default">
     <nav class="denhaag-anchor-navigation" aria-labelledby="secondary-navigation">
-      <ul id="secondary-navigation" class="denhaag-anchor-navigation__list">
+      <ol id="secondary-navigation" class="denhaag-anchor-navigation__list">
```

Comment:
This is a unnecessary change.

Reply:
I edit this one because it refers to content on the same page which is ordered.